banking-app/Makefile

```diff
@@ -40,7 +48,7 @@ test-mccf: build ## 🧪 Test the Banking Application in a Managed CCF environme
 # This is used in the demo scripts
 start-host: build ## 🏁 Start the CCF Sandbox for the demo
 	@echo -e "\e[34m$@\e[0m" || true
-	@/opt/ccf_virtual/bin/sandbox.sh --js-app-bundle ./dist/ --initial-member-count 3 --initial-user-count 0 --constitution-dir ./constitution
+	@/opt/ccf_virtual/bin/sandbox.sh --js-app-bundle ./dist/ --initial-member-count 3 --initial-user-count 0 --constitution-dir ./governance/constitution
```

Comment:
Raising a question here but it seems this value was already "wrong"
Don't we need users in Banking sample? If I'm not mistaken we need 2 for the application flow, therefore the sandbox script should include `--initial-user-count 2`

Reply:
I've double-checked in our repo.
`test_sandbox.sh` general script starts the sandbox with 2 users already
`banking-app/setup_governance.sh` adds 2 users manually during network setup

Therefore, `make start-host` should consider those users to allow a step-by-step flow. 
we just need to include `--initial-user-count 2`

Reply:
Updated to 2